packages/editor/src/hooks/align.js

```diff
@@ -23,6 +23,9 @@ import { BlockControls, BlockAlignmentToolbar } from '../components';
  * @return {Object}          Filtered block settings
  */
 export function addAttribute( settings ) {
+	if ( has( settings, [ 'attributes', 'align' ] ) ) {
```

Comment:
Can we add a test case for this, and ideally an inline code comment explaining its purpose?

Reply:
Do we expect `settings` to always be non-undefined?

- If yes, we don't need to incur Lodash's extra nullish check on `settings.attributes`, and instead express this as `has( settings.attributes, [ 'align' ] )`
- If no, then we have issues below where we assign a new `settings.attributes` on what we're arguing to be a potentially `undefined` settings object

Reply:
Hi @aduth thank you for catching this. This change is in no way related to this PR. I noticed that during the most recent rebase a commit of other branch was added here (I probably missed some command) the commit was removed from here. But I will apply the feedback on the correct place https://github.com/WordPress/gutenberg/pull/9338.